crates/runtime/examples/crdt.rs

```diff
@@ -0,0 +1,237 @@
+#![allow(unused_crate_dependencies, reason = "Not actually unused")]
```

Comment:
This example should be deleted and we should make sure that the kv value example stays the same while workung with this new functionality.

Reply:
I don't think this example should be deleted, as we do need to provide an example of hierarchical CRDT-based data, and that's what this shows modelled out. Otherwise, we don't have anything like this illustrated.

I don't disagree that the other examples should be updated as necessary.

Reply:
Im fine with keeping it, but main priority should be making only peers and kv store work with crdts.